build.rs

```diff
@@ -0,0 +1,7 @@
+fn main() {
+    // Build the CSS on-demand if we're running a release-ish build here (as determined by Cargo)
+    if std::env::var("PROFILE").is_ok_and(|v| v == "release") {
+        oranda_generate_css::build_css("oranda-css/dist/").unwrap();
```

Comment:
Will this work right if you run `cargo build` from a weird dir?

Reply:
yeah, `build.rs` runs from the cwd of the Cargo.toml, it seems